scipy/stats/tests/test_distributions.py

```diff
@@ -913,6 +913,92 @@ def test_sf(self):
         expected = (scale/x)**b   # 2.25e-18
         assert_allclose(p, expected)
 
+    @pytest.mark.filterwarnings("ignore:invalid value encountered in "
```

Comment:
I added this because a runtimewarning is brought up every time you fit some data with pareto.fit's super method. I am fairly sure it is not related to any changes here as the same issue happens when doing pareto.fit(...) on the master branch.

Reply:
@WarrenWeckesser this is the warning I was talking about. Since it happens on master and is unrelated to the changes here, @swallan will just open an issue.